client/src/components/products/EditProduct.js

```diff
@@ -0,0 +1,500 @@
+import React, { Component } from 'react';
+import DateFnsUtils from "@date-io/date-fns";
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker
+} from "@material-ui/pickers";
+import axios from "axios"
+import { makeStyles } from "@material-ui/core/styles";
+import {
+  TextField,
+  Button,
+  FormControl,
+  Grid,
+  InputAdornment,
+  Radio,
+  RadioGroup,
+  FormLabel,
+  FormControlLabel,
+  Select,
+  InputLabel,
+  MenuItem
+} from "@material-ui/core";
+import GoogleMapsInput from "./GoogleMapsInput";
+
+export default class EditProduct extends Component {
+  state = {
+    title: "",
+    description: "",
+    imageUrl: [],
+    brand: "",
+    category: "",
+    quantity: 0,
+    price: 0,
+    currency: "EUR",
+    tags: [],
+    company: "",
+    location: {
+      latitude: 52.52,
+      longitude: 13.405,
+      city: "",
+      address: "",
+      country: ""
+    },
+    markerPosition: {
+      lat: 52.52,
+      lng: 13.405
+    },
+    availability: null,
+    warrantyUntil: null,
+    condition: "",
+    isSold: false,
+    createdAt: null
+  };
+
+  getData = () => {
+    const id = this.props.match.params.id;
+    // console.log(`id`, id)
+
+    axios
+      .get(`/api/products/${id}`)
+      .then(response => {
+        this.setState({
+          title: response.data.title,
+          description: response.data.description,
+          imageUrl: response.data.imageUrl,
+          brand: response.data.brand,
+          category: response.data.category,
+          quantity: response.data.quantity,
+          price: response.data.price,
+          currency: response.data.currency,
+          tags: response.data.currency,
```

Comment:
This should be `response.data.tags`. Will be done once tags > checkbox is enabled.